src/redux/rockets/rockets.js

```diff
@@ -0,0 +1,22 @@
+const GET_ROCKETS = 'spaceTravelers/rockets/GET_ROCKETS';
+const RESERVE_ROCKET = 'spaceTravelers/rockets/RESERVE_ROCKET';
+const URL = 'https://api.spacexdata.com/v3/rockets';
+
+const reducer = (state = [], action) => {
+  switch (action.type) {
+    case GET_ROCKETS:
+      return action.state;
+    case RESERVE_ROCKET:
+      return action.state;
+    default:
+      return state;
+  }
+};
+
+export const loadRockets = () => async (dispatch) => {
+  const res = await fetch(URL);
+  const state = await res.json();
+  dispatch({ type: GET_ROCKETS, state });
+};
```

Comment:
This is just an optional comment 🤓 it's considered a good practice to fetch data using the familiar try/catch syntax in order to handle possible errors. And if you want to go a step beyond, create an action for the error and handle the type on your reducer.